packages/@aws-cdk/cfn-include/build.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * This build file has two purposes:
+ *   1. It adds a dependency on each @aws-cdk/aws-xyz package to this package,
+ *     similarly to how deps.js does for decdk.
+ *   2. It generates the file cfn-types-2-classes.json that contains a mapping
```

Comment:
Can you upload a copy of the generated file to the PR so I can take a look at it?

Reply:
Sure: https://gist.github.com/skinny85/4a22dc78796afffc8716e34026b81334